### PR TITLE
Pharokka update

### PR DESCRIPTION
**Pull request by paulzierep**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

Update tool to use data table with version column

### CODE REVIEW

File: tools/pharokka/tool-data/pharokka_db.loc.sample [hunk 1]
```diff
@@ -2,11 +2,11 @@
 #to use a pharokka DB folder. The pharokka_db.loc 
 #file needs this format (longer white space is the TAB character):
 
-#<unique_build_id>		<dbkey>		<display_name>		<DB_folder_path>
+#<unique_build_id>  <dbkey> <version>   <display_name>  <DB_folder_path>
```
Comment (bgruening) [inline tools/pharokka/tool-data/pharokka_db.loc.sample:5]: is that on par with the DM?

Comment (paulzierep) [reply tools/pharokka/tool-data/pharokka_db.loc.sample:5]: yes:
https://github.com/galaxyproject/tools-iuc/blob/main/data_managers/data_manager_pharokka_database_fetcher/tool_data_table_conf.xml.sample#L1-L7

Comment (bernt-matthias) [reply tools/pharokka/tool-data/pharokka_db.loc.sample:5]: As far as I know we can't change data tables, ie their columns...

Comment (paulzierep) [reply tools/pharokka/tool-data/pharokka_db.loc.sample:5]: Now that we modified the table on EU to make the tool work, we should probably leave it as is atm and only work on the DM to match that table... is that a correct assumption ?
@bernt-matthias what would the approach if something in the table would need to be changed, i.e. if an updated tool requires more data table columns ?

Comment (bernt-matthias) [reply tools/pharokka/tool-data/pharokka_db.loc.sample:5]: I guess: create a new data table. Then admins may move old entries to the new data table and create new entries in the new column.

As far as I know the behavior of Galaxy in the case is completely undefined. I guess in the best case this "just" breaks old tool versions (which will use the content of the version column as display name and the display name as path). If new columns are added in the middle it definitely breaks things. Not sure what happens if columns are appended. 

Not sure if adding a new column to the end might work (in the future https://github.com/galaxyproject/galaxy/issues/16260).

**Pull request merged**